README.md

```diff
@@ -306,6 +306,24 @@ Functions that have `groupedData` as parameter, means that `groupedData` is a JS
   - `continent`: Continent name that came from the GeoJSON map file.
 - `longitudeKey:` _(string, default: 'longitude')_ Column name of the longitude (from the dataset).
 - `latitudeKey:` _(string, default: 'latitude')_ Column name of the latitude (from the dataset).
+- `defaultScaleTo:` _(object, default:null)_ Default zoom state for the map based on either Country code or Latitude/Longitude point.
+
+  - `defaultScaleTo.scaleType:` _(string, default:null)_ Type of a zoom/scale on the map. This only accepts two types: 'lat/lng' for a zoom based on latitude and longitude points or 'country' for a zoom based on specific country code(iso_a3 code).
+  - `defaultScaleTo.values` _(object, default:null)_ Option values for the zoom state on the map based on 'scaleType' property. If 'scaleType' is set to 'lat/lng' then its 'values' object should contain: 'latitude','longitude' and 'scale' properties. If 'scaleType' is set to 'country' then its 'values' object should contain: 'iso_a3' property.
+
+    _Example:_
+
+    ```js
+    defaultScaleTo: {
+      scaleType: 'lat/lng',
```

Comment:
`coordinates` would be a better naming here.

Reply:
Done, scale type changed from 'lat/lng' to 'coordinates' now.

For France country selection issue,
This is happening because the territory of this country 'French Gunna' and some other island nations(like 'Martinique') in South America are also technically considered as France, and currently when 'France' is selected as a country on the map it's selecting all those territory as well and centering the map based on all those territories. So, I don't think we have any other options for this right now since it's technically displaying all France territory on the map. We can enhance data on 'countries.json' to get a better look for this type of territories though. But there is no any logical or code related issue here.

<img width="1140" alt="Screenshot 2024-11-24 at 15 30 26" src="https://github.com/user-attachments/assets/75dbd7a9-2ff9-4280-ad50-281079b43737">


For map scale issue,
I've updated this issue, and should be working properly now. For scale type 'country', I've disabled zoom-out functionality because the selected country will be the center of focus when selected. One can still zoom-in and translate the map in order to view specific point for currently selected country if needed.

For scale type 'coordinates', I've disabled both zoom and translate functionality as both the scale value and fixed coordinate points for zoom are passed from outside and so that map view stays focused to that one specific point based on those passed values.

Let me know if you have any further suggestions on this.

Reply:
Ah, good catch! I thought it was a bug, but it's actually a feature for the zoom around France :)